pkg/ffmpeg/testdata/README.md

```
@@ -39,6 +39,46 @@ ffmpeg -y -f lavfi -i color=c=green:s=100x100:d=1 -frames:v 1 -q:v 5 \
 
+## video_with_movtext_subtitle.mp4
+
+A synthetic mp4 with a 0.5 s 160x120 H.264 video, a 0.5 s AAC audio track, and
+a single mov_text subtitle ("Hello world"). The matroska muxer rejects
+mov_text outright (`Subtitle codec mov_text (94213) is not supported.`), so
+this fixture is used to regression-test the conversion of mov_text to a
+matroska-compatible subtitle codec on the way through the transcoder.
+
+Generation command:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)